cpp/ql/src/experimental/Likely Bugs/OverrunWriteProductFlow.ql

```diff
@@ -134,16 +134,34 @@ module StringSizeConfig implements ProductFlow::StateConfigSig {
 
 module StringSizeFlow = ProductFlow::GlobalWithState<StringSizeConfig>;
 
+int getOverflow(
+  DataFlow::Node source1, DataFlow::Node source2, DataFlow::Node sink1, DataFlow::Node sink2,
```

Comment:
Because the flow state is part of the `PathNode`'s list of columns. So if we changed the `Node`s to be `PathNode`s we'd have a `getOverflow` row for each flow state (and so the `max` in the `select` wouldn't do anything).

Reply:
I'm not sure if that's necessary. I'm happy with this as it is.

Reply:
Oh well. I've opened a PR with the changes now 😂.